shell/browser/net/electron_url_loader_factory.cc

```diff
@@ -671,20 +670,11 @@ void ElectronURLLoaderFactory::StartLoadingHttp(
       request->method != net::HttpRequestHeaders::kHeadMethod)
     dict.Get("uploadData", &upload_data);
 
-  auto* browser_context = ElectronBrowserContext::GetDefaultBrowserContext();
-  v8::Local<v8::Value> value;
-  if (dict.Get("session", &value)) {
-    if (value->IsNull()) {
-      browser_context = ElectronBrowserContext::From(
-          base::Uuid::GenerateRandomV4().AsLowercaseString(), true);
-    } else {
-      gin::Handle<api::Session> session;
-      if (gin::ConvertFromV8(dict.isolate(), value, &session) &&
-          !session.IsEmpty()) {
-        browser_context = session->browser_context();
-      }
-    }
-  }
+  gin::Handle<api::Session> session;
+  auto* browser_context =
+      dict.Get("session", &session) && !session.IsEmpty()
```

Comment:
If someone passes `null` (or some other value that's not a Session), then `session.IsEmpty()` will be `true`, correct? In other words, existing code will fall back to the default browser context rather than crashing?

Reply:
Yes, that's correct. It will default back to the default session. :+1: